graphql/src/adapters/logger_provider/basic.py

```diff
@@ -0,0 +1,25 @@
+import logging
+from typing import Any, MutableMapping
+
+from src.use_cases.adapter_interfaces.logger_provider import LoggerProviderInterface
+
+logger = logging.Logger("graphql")
+
+
+class BasicLoggerProvider(LoggerProviderInterface):
+    def get_logger(self, request_id: str) -> logging.Logger:
+        """
+        Args:
+            request_id: A unique identifier to track requests.
+
+        Returns: A logger for use in code.
+        """
+
+        # noinspection PyTypeChecker
+        return GuidLoggerAdapter(logger, {'request_id': request_id})
+
+
+class GuidLoggerAdapter(logging.LoggerAdapter):
```

Comment:
Reintroduced logger parameters for request-tracking. If this is still disallowed, it will need to be addressed via https://bugs.earthdata.nasa.gov/browse/ORCA-588